src/ConfigDefinition.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
je to vůbec potřeba, když to nemá žádnej config?

Reply:
nema, ale nechtel jsem to mazat, treba se tam nejakej config objevi